inventory/byo/hosts.ose.example

```diff
@@ -785,6 +785,25 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 #
 #etcd_ca_default_days=1825
 
+# Upgrade Control
+#
+# By default nodes are upgraded in a serial manner one at a time. You may
+# choose to upgrade either a specified number of nodes at one time or you
+# may specify a percentage of nodes to upgrade at once. If you're upgrading
+# more than one node at a time you can specify the maximum percentage of failure
+# within the batch before the upgrade is aborted.
+#
+# Be careful if you set openshift_upgrade_nodes_serial to a percentage and your
+# node counts will change. Also be mindful if you have a failure you should
+# return the failed node to service so that you don't exhaust your cluster
+# capacity.
+#
+# The following configuration would abort if two nodes in a batch of four failed.
+#openshift_upgrade_nodes_serial=4
```

Comment:
Outside of these examples, should we indicate the defaults of these variables? A standard practice is to have commented variables defined to their defaults.

```
#openshift_upgrade_nodes_serial=1
#openshift_upgrade_nodes_max_fail_percentage=0
```